src/lesson05/RealCalculator.java

```diff
@@ -0,0 +1,24 @@
+package lesson05;
+
+public class RealCalculator {
+
+    public static void main(String[] args) {
+        Calculator<Double, Character> calculator = (Double a, Double b, Character c) ->
+                c == ('+') ? (a + b) :
+                        c == ('*') ? a * b :
+                                c == ('/') ? a / b :
+                                        c == ('-') ? (a - b) :
+                                                c == ('v') ? Math.pow(a, 1 / b) :
+                                                        c == ('^') ? Math.pow(a, b) :
+                                                                c == ('&') ? Math.pow((a + b) / a + 117, b) : 0.0d;
```

Comment:
better make user informed that there is no such operator (with custom exception, not printing)